.github/workflows/build_group.yml

```
@@ -0,0 +1,35 @@
+# Build and test for the private repository for TUDelft Radar group
+# Only test on PR for dev branch, not triggered for draft PR
+
+name: Build and pytest
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches: [ "dev" ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest']
+        python-version: ["3.10", "3.11", "3.12"]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install build
+        python -m pip install ".[dev,docs,demo]"
```

```suggestion
        python -m pip install .[dev]
```

---

```suggestion
        python -m pip install ".[dev,docs]"
```

Thanks Sarah. I will do the dev and docs sincefor most of time it would be needed by the developers